ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLHttpService.java

```diff
@@ -285,19 +286,19 @@ private void handleGraphQLRequest(final RoutingContext routingContext) {
       final Map<String, Object> variables;
       final HttpServerRequest request = routingContext.request();
 
-      switch (request.method()) {
-        case GET:
+      switch (request.method().name()) {
```

Comment:
why switching to string when enum is safer?

Reply:
Sadly, they are not Enums, but rather static instances of HttpMethod :(